src/xml_parsing.cpp

```diff
@@ -989,6 +989,24 @@ void addNodeModelToXML(const TreeNodeManifest& model,
     element->InsertEndChild(description_element);
   }
 
+  for (const auto& metadata : model.metadata)
```

Comment:
The generated XML should be

```XML
<MetadataFields>
  <Metadata name="foo" value="hello"/>
  <Metadata name="bar" value="42"/>
</MetadataFields>

Reply:
Done! https://github.com/BehaviorTree/BehaviorTree.CPP/pull/730/commits/96910f265786458ac2f1b33a9f3d1a3adaf26473